packages/react-lowcode/src/codegen/app.ts

```diff
@@ -27,46 +30,98 @@ export function generatePages(inputSourceCode: string, io: CodeRW & CodeDir, opt
                 }))
             }
 
-            let context = {uiFramework: UiFramework.MaterialUI, formatter: Formatter.None, index: {tableType: TableType.BasicTable, height: "400px"}};
-            
-            const generator = new AppGenerator(context, entity)
-            const page = generator.generateListComponent(/* TODO entity / type name should be input - not in context */)
+            const entityListComponentPageName = getListPageComponentName(entity)
+            const listPageFilePath = `src/components/${entityListComponentPageName}.tsx`
+            const moduleRouteUri = `app${getPluralizedEntityName(entity.getName())}`
```

Comment:
for example `appcustomers` - seems we are missing a slash, aren't we? `app/customers`